src/scenes/levelend-scene.ts

```diff
@@ -228,7 +232,11 @@ export class LevelEndScene {
       // pass same data as level is same
       this.switchToGameplayCB(gamePlayData, "LevelEnd");
     }
-    if (this.nextButton.onClick(x, y) && this.starCount >= 2) {
+    if (
+      this.nextButton.onClick(x, y) &&
+      this.starCount >= 2 &&
+      this.isLastLevel
```

Comment:
this.nextButton.onClick(x, y) move this as the last condition so it won't fore the this.IsLastLevel is false.

Reply:
Hi @bernhardccodev please check, again. thanks